contracts/Core/RewardManager.sol

```diff
@@ -127,37 +127,37 @@ contract RewardManager is Initializable, ACL, RewardStorage {
 
         // no rewards if last epoch didn't got revealed
         if ((epoch - epochLastRevealed) != 1) return;
-        uint256[] memory mediansLastEpoch =
-            blockManager.getBlockMedians(epochLastRevealed);
-        uint256[] memory lowerCutoffsLastEpoch =
-            blockManager.getLowerCutoffs(epochLastRevealed);
-        uint256[] memory higherCutoffsLastEpoch =
-            blockManager.getHigherCutoffs(epochLastRevealed);
 
-        if (lowerCutoffsLastEpoch.length > 0) {
+        Structs.Block memory blockLastEpoch = blockManager.getBlock(
```

Comment:
This change was made to keep givePenalties and giveRewards structure consistent.